ui/app/models/job.js

```diff
@@ -246,7 +244,7 @@ export default Model.extend({
     const id = payload.Name;
 
     this.set('plainId', id);
-    this.set('id', JSON.stringify([id, namespace]));
+    this.set('_idBeforeSaving', JSON.stringify([id, namespace]));
```

Comment:
You [can’t change the id after the record has been created](https://discordapp.com/channels/480462759797063690/486549196837486592/601083103364251659), so this instead saves to a private-y property for later use in the job adapter.